analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -74,6 +74,8 @@ import org.ossreviewtoolkit.utils.searchUpwardsForFile
 private const val DEFAULT_SERVICE_INDEX_URL = "https://api.nuget.org/v3/index.json"
 private const val REGISTRATIONS_BASE_URL_TYPE = "RegistrationsBaseUrl/3.6.0"
 
+private val VERSION_RANGE_CHARS = charArrayOf('[', ']', '(', ')', ',')
```

Comment:
Should we handle also the `*` and `-` from the "Floating version resolutions" section in the linked docs?

Reply:
I believe not. The aim was to only remove "outer" characters here, to the to the highest (or lowest) version.

Reply:
Also, I've added a TODO now to support floating versions.